### PR TITLE
Typo Fixes

### PR DESCRIPTION
**Pull request by w01fS**
Found certain wrong spellings.
Additionally, dotfiles have been placed in the tilde block to highlight them from the surrounding text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1103,7 +1103,7 @@ Bear in mind that with the introduction of the new V8 engine alongside the new E
 
 ## ![✔] 8.3. Remove development dependencies
 
-**TL;DR:** Althoug DevDepencies are sometimes needed during the build and test life-cycle, eventually the image that is shipped to production should be minimal and clean from development depdencies. Doing so gurantess that only neccessary code is shipped and the amount of potnetial attacks (i.e. attack surface) is minimized. When using multi stage build (see dedicated bullet) this can be achieved by installing all dependencies first and finally running 'npm ci --production'
+**TL;DR:** Although Dev-Dependencies are sometimes needed during the build and test life-cycle, eventually the image that is shipped to production should be minimal and clean from development dependencies. Doing so guarantees that only necessary code is shipped, and the amount of potential attacks (i.e. attack surface) is minimized. When using multi-stage build (see dedicated bullet) this can be achieved by installing all dependencies first and finally running 'npm ci --production'
```
Comment (kevynb) [inline README.md:1106]: We should not add a comma before an `and`:  `necessary code is shipped and the amount`

Comment (w01fS) [reply README.md]: Done.
Please let me know if any other change is required.

**Comment by kevynb** [comment]
Sorry @w01fS we reordered all the docker bullet-points before merging to master. Would you mind editing your PR accordingly because now your changes are on the wrong BP? 
If you don't I understand and we can just close it :)

**Comment by w01fS** [comment]
@kevynb Changes done.

Please let me know if something else is required.

**Review by kevynb — APPROVED** [review]

**Comment by kevynb** [comment]
Thanks for bearing with us @w01fS!

@all-contributors please add @w01fS for content

**Pull request merged**